pkg/daemon/ceph/osd/volume.go

```diff
@@ -511,7 +513,9 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 func (a *OsdAgent) initializeDevicesRawMode(context *clusterd.Context, devices *DeviceOsdMapping) error {
 	baseCommand := "stdbuf"
 	cephVolumeMode := "raw"
-	baseArgs := []string{"-oL", cephVolumeCmd, cephVolumeMode, "prepare", "--bluestore"}
+	storeFlag := fmt.Sprintf("--%s", a.storeConfig.StoreType)
```

Comment:
```suggestion
	storeFlag := fmt.Sprintf("--%s", c.spec.Storage.GetOSDStore())
```
?

Reply:
This function runs inside the OSD prepare pod. There is no reference to the cluster spec inside the osd prepare pod. Can you check again if I missed something?

Reply:
right, there is no reference to cluster spec but could we add it similar to NetworkSpec, or calling `GetOSDStore()` doesn't make sense here?

Reply:
I think it does not. We have added the storeType from the spec to the prepare pod environment variable.  The prepare pod is reading it from the env variable now.